### ISSUE TITLE
The merging process end with error. 

### ISSUE THREAD

**Issue by kabilsharma** [Mbucari/GEHistoricalImagery#3]
![{C3AAB4B7-75B1-4665-96C7-52A8255C4F88}](https://github.com/user-attachments/assets/b205cd03-3d9d-4191-9eb7-71aa8e9da591)

**Comment by kabilsharma** [comment]
dotnet run --project src\GEHistoricalImagery\GEHistoricalImagery.csproj -- download --lower-left "22.4647,88.2303" --upper-right "22.6114,88.3639" --output "D:\Kolkata_2020-05-12.tif" --zoom 20 --date "2020/05/12" --parallel 4

**Comment by Mbucari** [comment]
@kabilsharma Thanks for providing the command line you used. Currently, GEHistoricalImagery compresses the geotiff as a YCBCR jpeg. Using that format saves space, but it's limited to a maximum of 65,535 pixels tall/wide. The area of interest in your region is 99,619x109,387 pixels, far exceeding that limit.

Unfortunately, the packages I'm using do not support Jpeg-XL, so the next best available compression is DEFLATE, which yields TIFFs that are about 9x larger than with the JPEG compression. Right now your only option is to split up the area into smaller rectangles. 4 smaller images, each ~50,000x55,000 will do the trick.

**Comment by Mbucari** [comment]
OK, I found a better approach. GeoTiffs will now be written with Tiles of 256x256 pixel Jpegs. This removes the size limit and keeps the tiffs small.

[Fixed in the latest release](https://github.com/Mbucari/GEHistoricalImagery/releases/latest).

**Comment by kabilsharma** [comment]
Thanks i figured out this and was breaking coordinates into small boxes.

**Comment by Mbucari** [comment]
I did not know about ECW. I did look into it, and I'll need to get a license to write files in the ECW format. I do not foresee myself doing that.

By the way, are you familiar with the built-in warping capabilities? The default spatial reference is WGS84, so the imagery get more vertically compressed the further away from the equator it is. For you example, consider a projected coordinate system such as [UTM Zone 45 North](https://epsg.io/32645).

```console
GEHistoricalImagery download --lower-left "22.4647,88.2303" --upper-right "22.6114,88.3639" --output ".\Kolkata_2023-04-16-UTM.tif" --zoom 20 --date "2023/04/16 --target-sr https://epsg.io/32645.wkt2"
```
